### PR TITLE
Updates to the 'Importing modules into Pantheon' module

### PR DESCRIPTION
**Pull request by RichardHoch**
Resolves https://projects.engineering.redhat.com/browse/FCCEUD-47

### CODE REVIEW

**Comment by RichardHoch** [comment]
@redhataccess/pantheon-developers Please review for technical completeness and accuracy.

**Comment by benradey** [comment]
@RichardHoch What you have here looks great - I might consider adding a blurb about the sync happening in the background - so just because you are immediately redirected in the UI, doesn't mean that all your files will be immediately available. Up to you if you think that is worth including.

**Comment by RichardHoch** [comment]
@benradey -- please give the last paragraph ("Note that...")  a quick lookover before I ask for peer review. Thanks.

**Comment by benradey** [comment]
@RichardHoch My apologies for keeping you waiting, but the update looks perfect to me - thanks!

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+//* You are assigned a publisher or author role in {ProductShortName}.
+* You are logged in.
+* You obtained the Git repository URL for the {ContentTerm} you want to import (For example, https://github.com/openshift/openshift-docs).
+
+.Procedure
+
+. Navigate to the link:{LinkToGitImportPage}[Git Import page].
+. Enter the Git repository URL for the {ContentTerm} you want to import.
+. By default, this procedure imports the "master" branch of the {ContentTerm} repository, but you can enter a specific branch to import instead if you prefer.
```
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc:18]: I don't understand this line. How do you import master? How do you import a branch?

File: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+. Click *Submit*.
+. Click *Yes* to confirm and return to the link:{LinkToSearchPage}[Search page], or *No* to remain on the Git Import page.
++
+Note that it takes {ProductShortName} some time to synchronize its database and the UI following the import. Therefore your files may not be immediately available when you are redirected in the UI,
```
Comment (stoobie) [inline pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc:22]:
```suggestion
Note that {ProductShortName} takes time to synchronize its database and the GUI following the import. Therefore your files might not be immediately available when you are redirected in the GUI,
```
Looking for more brevity. Also, don't use "may", it's ambiguous.

**Review by stoobie — CHANGES_REQUESTED** [review]
Two comments, otherwise looks good!

**Review by stoobie — APPROVED** [review]
LGTM

**Comment by stoobie** [comment]
@benradey or @redhataccess/pantheon-developers This PR is ready to be merged.

**Comment by RichardHoch** [comment]
@benradey or @redhataccess/pantheon-developers -- please merge this PR.

**Review by benradey — APPROVED** [review]

**Pull request merged**